Bottom: 86aba3185d95e209ea12c017663afbc8fbb46c12
Top:    ac1ad4a786a593bcc379852f8b5dd5ba45327c80
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:41:57 +0000

Add reversed orientation field to face3d structure


---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 4838c43..95232a6 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
